README.md

```diff
@@ -17,13 +17,13 @@ This is useful for:
 
 - [Microsoft Visual Studio Code](https://code.visualstudio.com)
 - [Hyper](https://hyper.is/)
-- [Upterm](https://github.com/railsware/upterm)
 - [Script Runner](https://github.com/ioquatix/script-runner) for Atom.
 - [Theia](https://github.com/theia-ide/theia)
 - [FreeMAN](https://github.com/matthew-matvei/freeman) file manager
-- [atom-xterm](https://atom.io/packages/atom-xterm) - Atom plugin for providing terminals inside your Atom workspace.
```

Comment:
As in https://github.com/xtermjs/xterm.js/pull/2723 I think we should leave this for now.

Reply:
OK.